test/express-redis/tests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:3.0
+
+RUN bundle config --global frozen 1
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+
+COPY . .
```

Comment:
@tombruijn There is a point to copying the Gemfile and its lock first, then the rest of the application. It allows the Docker cache to reuse the `RUN bundle install` layer if the Gemfile and its lock haven't changed.

Reply:
Ok, but that's only locally. On Semaphore it doesn't cache Docker layers.

If we use a volume we could cache the bundle directory on the CI and locally, so it doesn't need to download all gems all the time. We'd take small performance hit because it's mounted on a volume, but not as bad as on local macOS machines.